pom.xml

```diff
@@ -15,6 +15,7 @@
         <bitbucket.version>5.5.9</bitbucket.version>
         <cloverVersion>4.3.1</cloverVersion>
         <disableTestInjection>false</disableTestInjection>
+        <google.guava.libversion>29.0-jre</google.guava.libversion>
```

Comment:
part of the ticket was to work out which was the best caching solution, can you elaborate on why we picked Guava? We _know_ guava will give us headaches if we're not extremely careful, so I'm keen to hear why we didn't go with the cache built into okhttp for example

Reply:
I tried importing it, and given it looks like we're actually already using Guava in a few places, it seemed like it would be okay to use. With the one failing integration test excepted, I'm not sure what headaches it's caused. That said, are there other issues with this solution we can't see in this PR? If so, happy to close and use a different solution (okhttp sounds good)